docs/manage/partitions/flex/estimate-scan-data.md

```diff
@@ -34,6 +34,18 @@ As Sumo Logic charges for each data scan, scheduling searches can help mitigate
 Daily scan data will not be available if scheduled search runs only on selected days and not daily.
 :::
 
+## Scan estimates in SLO
+
+When creating or modifying a log-based SLO, you can view the Daily scan and Yearly scan data information based on the SLO details by clicking the meter icon <img src={useBaseUrl('/img/partitions-data-tiers/flex-pricing/meter-icon.png')} alt="meter-icon" width="25" /> on top of the SLO right side panel.
+
+Estimate the scan for different types of log-based SLOs:
+
+- **Request-Based SLOs**. For request-based SLOs, data is scanned once per minute and stored in SLO storage. Any late-arriving data is accounted for in a corrective fashion and appended to the SLO storage.
+
+- **Window-Based SLOs**. For window-based SLOs, data is scanned multiple times per minute. Due to technical reasons, late-arriving data cannot be accounted for, unlike request-based SLOs. Because of this, the SLO results are stored in SLO storage with some delay (typically 1 hour). To display up-to-date SLI values across our various SLO dashboards, we need to scan the last 1 hour of data multiple times.
```

Comment:
```suggestion
- For **Request-based SLOs**, data is scanned once per minute and stored in SLO storage. Any late-arriving data is accounted for in a corrective fashion and appended to the SLO storage.
- For **Window-based SLOs**, data is scanned multiple times per minute. Due to technical reasons, late-arriving data cannot be accounted for, unlike request-based SLOs. Because of this, the SLO results are stored in SLO storage with some delay (typically 1 hour). To display up-to-date SLI values across our various SLO dashboards, we need to scan the last 1 hour of data multiple times.
```